src/test/java/sdmExam/BoardShould.java

```diff
@@ -1,75 +1,113 @@
 package sdmExam;
 
+import org.junit.jupiter.api.DynamicTest;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.List;
 import java.util.NoSuchElementException;
+import java.util.stream.Stream;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 public class BoardShould {
-    private final Board board = new Board();
+    private final static Board board = new Board();
```

Comment:
Why static?

Reply:
It was a warning from intellij, we can revert if it conflicts with other part

Reply:
Ok, but what did the inspector tell?

Reply:
Wait remembered now, I had to do it because I am using @MethodSource("Method")

Those "Methods" which is static provide a stream for test purposes, without having board as static variable it gives error like this: `Non-static field 'board' cannot be referenced from a static context.`

I checked without having "Method" as static, it can't compile, I read that to use methods in @MethodSource they need to be static methods. That is the reason. Any idea if you want to make it not static?

Reply:
Ok. No, it's fine we can leave it static then.